docs/content/configuration/configuring_hostdefiner.md

```diff
@@ -0,0 +1,12 @@
+# Configuring the host definer
+
+Some of the parameters within the HostDefiner custom resource are configurable. Use this information to help decide whether the parameters for your storage system need to be updated.
+
+For more information about using the host definer, see [Using dynamic host definition](../using/using_hostdefinition.md).
+    
+|Field|Description|
+|---------|--------|
+|`prefix`|Adds a prefix to the hosts defined by the host definer.|
+|`connectivity`|Selects the connectivity type for the host ports.<br>Possible input values are:<br>- `nvme` for use with NVME over Fibre Channel connectivity<br>- `fc` for use with Fibre Channel over SCSI connectivity<br>- `iscsi` for use with iSCSI connectivity<br><br>By default, this field is blank and the host definer selects the first of available connectivity types on the node, according to the following hierarchy: NVMe, FC, iSCSI.|
+|`allowDelete`|Defines whether the host definer is allowed to delete host definitions.<br>Input values are `true` or `false`.<br>The default value is `true`.|
```

Comment:
Please add description that not just hostdefinition will be deleted but the host object from the storage.

Reply:
updated

Reply:
>is allowed to delete host definitions

this means definitions on storage. an "object" is just a general term that includes a "definition".
to me it reads as if the same thing is written twice.
AFAIK we **never** refer to the k8s hostdefinition with a space between "host" and "definition".
@matancarmeli7 ?

Reply:
updated in meeting with @matancarmeli7 and @roysahar

Reply:
IMO it is more important for the user to know if the definer is allowed to delete definitions on storage rather than to delete its internal `HostDefinition` k8s objects

Reply:
@oriyarde We decided to word this as follows based off of the following:
1. When describing the feature in [Using dynamic host definition](https://github.com/IBM/ibm-block-csi-driver/blob/docs_release-1.10.0/docs/content/using/using_hostdefinition.md) we state there that this is happening on the storage.
2. When talking about [configuration for PVC](https://github.com/IBM/ibm-block-csi-driver/blob/docs_release-1.10.0/docs/content/configuration/creating_pvc.md) we don't mention storage side each time.

@roysahar  @matancarmeli7  please feel free to expand on this or correct anything that I missed / misunderstood based on our conversation the other day. thanks

Reply:
1. true, though we didn't explicitly write about the definer **deleting** definitions on storage
2. a PVC is managed by the user, a `HostDefinition` is managed by the definer. in essence, the user shouldn't care much about the existence of the k8s `HostDefinition` object (unless they want to troubleshoot/verify actions from the k8s side)

the reason the `allowDelete` parameter exists is in case the user relies on the storage definitions for non-CSI usage, and manually deleting the k8s `HostDefinition` object doesn't make the definer delete the storage definitions, so what matters is the deletion on storage, not in k8s

Reply:
so should I just add "on the storage system" to the end of the sentence?

Reply:
not exactly, because we also use a different casing convention for each side. I suggest:
```
Defines whether the host definer is allowed to delete host definitions on the storage system.
```

Reply:
updated and can be found [here](https://github.com/IBM/ibm-block-csi-driver/blob/docs_release-1.10.0/docs/content/configuration/configuring_hostdefiner.md)

Reply:
>we also use a different casing convention for each side

so `HostDefinitions on the storage system` should be `host definitions on the storage system`

Reply:
From my conversation with @roysahar  and @matancarmeli7  we were talking about the HostDefinition file itself. is this no longer the case when talking about on the storage side?

Reply:
AFAIK it is (and it has always been) `HostDefinition` for the k8s side, and "host definition" for the storage side

Reply:
I updated. @roysahar  please confirm that this shoudl be "host definitions on the storage system" and _not_ "HostDefinitions on storage system"

Reply:
confirmed

Reply:
this is showing on https://github.com/IBM/ibm-block-csi-driver/pull/532/files#diff-b882cc376223979bdac6e43eb33e525baefc16e5d726e0cc879ca309206675f4